docs/snippets/about-revisions.md

```diff
@@ -0,0 +1,13 @@
+<!-- Snippet used in the following topics:
+- /docs/concepts/resources/revisions.md
+- /docs/serving/revisions/README.md
+-->
+Revisions are stateless Knative Serving resources that contain point-in-time snapshots of the application code and configuration for each change made to a Knative Service.
```

Comment:
```suggestion
Revisions are Knative Serving resources that contain point-in-time snapshots of the application code and configuration for each change made to a Knative Service.
```
I would remove the word stateless to add confusion.

Reply:
This came from the API spec https://github.com/knative/specs/blob/main/specs/serving/knative-api-specification-1.0.md#revision so if we remove it here it should also be updated there IMO?

Reply:
The spec is correct
>The Knative Revision represents a stateless, autoscaling snapshot-in-time of application code and configuration. 

Is saying that a revision is a representation of code and configuration of a stateless application. 
The thing that is stateless if the application code&config itself.

Reply:
Ah, OK. I think it's misleading how it's worded in the spec still, but I understand what you're saying now.

Reply:
What about these options?
1. Revisions represent point-in-time snapshots of the application code and configuration for each change made to a Knative Service.
2. Revisions represent point-in-time snapshots of the stateless application code and configuration for each change made to a Knative Service.